CHANGES.rst

```diff
@@ -1,7 +1,7 @@
 Changes
 =======
 
-Version 0.7.3 (UNRELEASED)
+Version 0.7.3 (2021-03-18)
 --------------------------
 - Adds workflow parameters validation to ``validate`` command.
```

Comment:
Adds validation of workflow input parameters to the ``validate`` command.

Reply:
Adds optional validation of workflow environment images to the ``validate`` command when `--environments`` option is used.

(hmm, a bit long... just wanted to find something similar to previous line, so that they would play nicely together...)